build.gradle

```diff
@@ -950,8 +948,6 @@ project(':core') {
     from(project(':streams:test-utils').configurations.runtime) { into("libs/") }
     from(project(':support-metrics-common').jar) { into("libs/") }
     from(project(':support-metrics-common').configurations.runtime) { into("libs/") }
```

Comment:
@cmccabe I'm thinking we can remove this too. The reasoning is that we don't need this in the broker classpath, but other projects still depend on `support-metrics-common` as a library. What do you think?

Reply:
Actually, maybe we don't need to do this since `kafka-run-class` does not include `support-metrics-common` in the classpath.